components/connectivity-adapter/cmd/main.go

```diff
@@ -1,16 +1,47 @@
 package main
 
-import "net/http"
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/kyma-incubator/compass/components/connectivity-adapter/internal/appregistry"
+	"github.com/kyma-incubator/compass/components/connectivity-adapter/internal/connector"
+	"github.com/kyma-incubator/compass/components/connectivity-adapter/pkg/health"
+	"github.com/pkg/errors"
+	"github.com/vrischmann/envconfig"
+)
+
+type config struct {
+	Address string `envconfig:"default=127.0.0.1:8080"`
+
+	AppRegistry appregistry.Config
+	Connector   connector.Config
+}
 
 func main() {
-	http.HandleFunc("/healthz", healthz)
-	err := http.ListenAndServe(":8080", nil)
-	if err != nil {
-		panic(err)
-	}
+	cfg := config{}
+	err := envconfig.InitWithPrefix(&cfg, "APP")
+	exitOnError(err, "while loading app config")
 
+	router := mux.NewRouter()
+	router.HandleFunc("/healthz", health.HandleFunc)
+
+	appRegistryRouter := router.PathPrefix("/app-registry/v1").Subrouter()
+	appregistry.RegisterHandler(appRegistryRouter, cfg.AppRegistry)
+
+	connectorRouter := router.PathPrefix("/connector/v1").Subrouter()
```

Comment:
Same as with the `appRegistryRouter` router, currently there is no such route. I would go with `/v1/applications` prefix.

Reply:
I agree 👍 Fixed